### PR TITLE
Dump output data to SQL files

### PR DESCRIPTION
**Pull request by kevincianfarini**
Doing conflict resolution with CSV output from this CLI was going to be challenging, if not impossible. This commit outputs SQL files which can be run to against a database to keep Member of Congress information up-to-date.

### CODE REVIEW

**Comment by EvanHahn** [comment]
How will this be used? To seed the database?

File: src/nativeTest/kotlin/org/climatechangemakers/parsecongress/DumpToSqlTest.kt [hunk 1]
```diff
@@ -0,0 +1,122 @@
+package org.climatechangemakers.parsecongress
```
Comment (EvanHahn) [inline src/nativeTest/kotlin/org/climatechangemakers/parsecongress/DumpToSqlTest.kt:1]: Could you add a test that shows how single quotes are escaped? I'm less concerned about SQL injection and more concerned about general head-scratching bugs.

File: src/nativeMain/kotlin/org/climatechangemakers/parsecongress/Sql.kt [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  values.joinInto(this, separator = ", ") { value ->
+    append(value.sqlEscape())
+  }
+
+  append(")")
+}
+
+private fun Any?.sqlEscape(): String = when (this) {
+  null -> "NULL"
+  is String -> "'$this'"
```
Comment (EvanHahn) [inline src/nativeMain/kotlin/org/climatechangemakers/parsecongress/Sql.kt:111]: Can we escape apostrophes here? I worry about things like "Martha's Vineyard" causing problems.

**Review by EvanHahn — APPROVED** [review]
LGTM, but I think we should still try to avoid possible SQL injection bugs. Partly for security, but mostly for reliability.

I could see using a query builder library to achieve this goal instead, which I suspect will be more reliable in the long run and also solve this issue. But building up strings is also fine.

File: src/nativeTest/kotlin/org/climatechangemakers/parsecongress/DumpToSqlTest.kt (hunk 1 above)
Comment (kevincianfarini) [reply src/nativeTest/kotlin/org/climatechangemakers/parsecongress/DumpToSqlTest.kt:1]: Done!

**Pull request merged**